### DISCUSSION TITLE
`ImportError: DLL load failed while importing pytransform: The specified procedure could not be found.`

### DISCUSSION THREAD

**Discussion by tmotagam** [dashingsoft/pyarmor#1208] [Q&A]
**Traceback (most recent call last):
   File "C:\Users\comp\Document\new\obf\hello.py"
ImportError: DLL load failed while importing pytransform: The specified procedure could not be found.
[6624] Failed to execute script 'hello' due to unhandled exception!**

Hello,
This is the error that i am facing when executing the exe that is made with pyarmor 6.8.1, pyinstaller 4.7 and python 3.9 in windows 10 32bit. I created the same exe in windows 10 64bit and it is working fine.

The pyarmor creates the exe without and issue but when i execute the said exe it exits with the above trackback and it is happening in windows 10 32bit.
Thanks.

**Comment by jondy** [comment]
Please check section "when things to wrong" in Pyarmor 7.x documentation to find pack solution.
